hack/generator/pkg/astmodel/object_type.go

```diff
@@ -92,6 +164,22 @@ func (objectType *ObjectType) Properties() []*PropertyDefinition {
 	return result
 }
 
+func (objectType *ObjectType) Property(name PropertyName) (*PropertyDefinition, bool) {
+	prop, ok := objectType.properties[name]
+	return prop, ok
+}
+
+// HasFunctionWithName determines if this object has a function with the given name
+func (objectType *ObjectType) HasFunctionWithName(name string) bool {
```

Comment:
I think you could rewrite this the same way `Property()` works:

```
// HasFunctionWithName determines if this object has a function with the given name
func (objectType *ObjectType) HasFunctionWithName(name string) bool {
	_, ok := objectType.properties[name]
	return ok
}
```

Reply:
Doh, I can